haskell/test/Test/Ligo/BaseDAO/Common.hs

```diff
@@ -138,8 +138,7 @@ instance ProposalMetadataFromNum Integer where
   proposalMetadataFromNum = fromIntegral
 
 instance ProposalMetadataFromNum ProposalMetadata where
```

Comment:
I wonder if this class is still necessary at all.

AFAIK it was intended to simplify the use because it was a `Map`, which is no longer the case, what do you think?